api/client/proto/types.go

```diff
@@ -73,7 +75,7 @@ func (req *HostCertsRequest) CheckAndSetDefaults() error {
 // CheckAndSetDefaults checks and sets default values.
 func (req *ListResourcesRequest) CheckAndSetDefaults() error {
 	if req.Namespace == "" {
-		return trace.BadParameter("missing parameter namespace")
+		req.Namespace = apidefaults.Namespace
```

Comment:
I don't think we should require namespace, from what i've seen, it seems to always default to `apidefaults.Namespace`

Reply:
This behavior does look more in line with other `CheckAndSetDefaults`